stable/elastalert/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: ElastAlert is a simple framework for alerting on anomalies, spikes, or other patterns of interest from data in Elasticsearch.
 name: elastalert
-version: 0.5.0
+version: 0.5.1
```

Comment:
new feature, so I'd say 0.6.0

Reply:
@davidkarlsen I was hesitating while changing the version ^^... Just set it to 0.6.0 ;)